borders_shadows/styles.css

```diff
@@ -0,0 +1,155 @@
+html, body {
+  height: 100%;
+}
+
+body {
+  display: flex;
+  margin: 0;
+  font: 100% / 1.5 sans-serif;
+  background: url("images/moon-bg.jpg") no-repeat fixed center / cover black;
+}
+
+.home {
+  color:hsl(0, 0%, 95%);
+  padding: 1rem;
+  /* border: 1px solid red; */
+  text-decoration: none;
+  display: inline;
+  position: absolute;
+}
+
+.home:hover {
+  text-decoration: underline;
+}
+
+.disclosure-alert {
+  background-color: hsl(0, 0%, 95%);
+  width: 85%;
+  max-width: 48rem;
+  margin: auto;
+  color: hsl(0, 0%, 20%);
+  border: 1px solid hsl(0, 0%, 0%);
+  border-radius: 1.5rem;
+  box-shadow: 0 0 0.5rem hsl(300, 40%, 5%),
+              0 6rem 4rem -2rem hsl(300, 40%, 5%),
+              0 0.5rem 6rem hsl(200, 40%, 30%),
+              0 0.5rem 10rem hsl(300, 40%, 30%);;
+}
+
+.disclosure-header {
+  background: linear-gradient(hsl(300, 50%, 20%), hsl(300, 50%, 10%));
+  padding: 2rem 0.5rem;
+  text-align: center;
+  color: hsl(300, 50%, 95%);
+  border-top: 1px solid hsl(300, 50%, 35%);
+  border-bottom: 1px solid hsl(300, 50%, 5%);
+  /*the first time we used border radius, the white of .disclosure-alert was actually showing up behind .disclosure-header. Adjusting for the border (2px or .125rem), we decreased the border-radius and the peaking color is gone. */
+  /* border-radius: 1.5rem 1.5rem 0 0; */
```

Comment:
Great work around, another alternative if you wanted to keep this original border-radius could be just to reduce the radius of the top left and right corners of the disclosure alert since those corners arent intended to be visible.

Reply:
Thanks so much for all these ideas and best practices! 😁